Module1Practice.R

```diff
@@ -29,10 +29,15 @@ library(stringr)
 
 # Hardwired inputs for now
 # set directory
-setwd("~/R_Projects/group_TADA")
+getwd()
+#setwd("~/R_Projects/group_TADA")
+setwd("C:/Users/cmulli01/Documents/GitHub/group_TADA")
 fn = "WQP_Sample_Raw-11-25-20.csv"
 coi = "Temperature, water"
-mypercentile = .3
+
+# mypercentile = unname(round(quantile(result, c(.85)),1))
```

Comment:
This is the code from the original file: mypercentile = unname(round(quantile(result, c(.85)),1)). It does not work to just use this code up front because the script has been converted to a function. So we are using a hardwired threshold instead. How would we go about converting this to a percentile or adding a percentile as an option? It would be nice to include the option to manually enter a threshold and an option to manually define the percentile.

Reply:
Fix my lame percentile setting.